CITATIONS.md

```diff
@@ -73,8 +73,12 @@
 - [EUKulele](https://github.com/AlexanderLabWHOI/EUKulele)
 
 - [CAT](https://github.com/dutilh/CAT)
+
```

Comment:
Should this empty line be here? You don't seem to have added one after transrate below.

Reply:
Looks like most of them have an empty line between link and full citation. I did neglect one for transrate, but the markdown omits them either way.

Reply:
OK.